### ISSUE TITLE
Multiple Typeos and other Issues in commit 62d382

### ISSUE THREAD

**Issue by narthollis** [madcowfred/evething#37]
In multiple places you have `)exclude(` instead of `).exclude(`
- thing/views/assets.py:83
- thing/views/contracts.py:38
- thing/views/industry.py:42
- thing/views/orders.py:92
- thing/views/transactions.py:81
- thing/views/wallet_journal.py:86

Additionaly, after fixing these types, the following error occures when viewing the assets page.

```
Traceback (most recent call last):

  File "/.../thingenv/local/lib/python2.7/site-packages/django/core/handlers/base.py", line 115, in get_response
    response = callback(request, *callback_args, **callback_kwargs)

  File "/.../thingenv/local/lib/python2.7/site-packages/django/contrib/auth/decorators.py", line 25, in _wrapped_view
    return view_func(request, *args, **kwargs)

  File "./thing/views/assets.py", line 96, in assets_summary
    apikeys__key_type=APIKey.CORPORATION_TYPE,

  File "/.../thingenv/local/lib/python2.7/site-packages/django/db/models/manager.py", line 155, in filter
    return self.get_query_set().filter(*args, **kwargs)

  File "/.../thingenv/local/lib/python2.7/site-packages/django/db/models/query.py", line 669, in filter
    return self._filter_or_exclude(False, *args, **kwargs)

  File "/.../thingenv/local/lib/python2.7/site-packages/django/db/models/query.py", line 687, in _filter_or_exclude
    clone.query.add_q(Q(*args, **kwargs))

  File "/.../thingenv/local/lib/python2.7/site-packages/django/db/models/sql/query.py", line 1271, in add_q
    can_reuse=used_aliases, force_having=force_having)

  File "/.../thingenv/local/lib/python2.7/site-packages/django/db/models/sql/query.py", line 1139, in add_filter
    process_extras=process_extras)

  File "/.../thingenv/local/lib/python2.7/site-packages/django/db/models/sql/query.py", line 1337, in setup_joins
    "Choices are: %s" % (name, ", ".join(names)))

FieldError: Cannot resolve keyword 'apikeys' into field. Choices are: alliance, character, contract, corporationstanding, corpwallet, division1, division2, division3, division4, division5, division6, division7, id, industryjob, journalentry, name, ticker, transaction
```

**Comment by madcowfred** [comment]
That's pretty much what I deserve for pushing completely untested changes at 3am. Should all be fixed now in cf592dd.

**Comment by narthollis** [comment]
Thanks for fixing this so quickly. I have pulled these changes and it all appears to be working.
